src/app/lib/logging.ts

```diff
@@ -24,12 +24,14 @@ export const time_this = async <T>(
     throw error;
   } finally {
     const end = performance.now();
-    // console.log(
-    //   util.inspect(
-    //     { ...log_fields, time: end - start },
-    //     { showHidden: false, depth: null, colors: !log_emission }
-    //   )
-    // );
+    if (!process.env.NEXT_PUBLIC_MUTE_QUERY_LOGGING) {
```

Comment:
lol thanks sorry I'd commented it out, I was having trouble finding logs that I was logging from components because every time I saved I'd get mountains of logs from this file.

nice that there is a flag here now